### DISCUSSION TITLE
Trojan found in pc version of this

### DISCUSSION THREAD

**Discussion by zerobest1123** [thatskymod/Sky-CotL-Scripts#94] [Q&A] [answered]
![image_2024-06-02_164516294](https://github.com/thatskymod/Sky-CotL-Scripts/assets/171288565/9506cbab-66d6-441f-b3e4-776bc5c4b4d4)

**Comment by zerobest1123** [comment]
false positive? or should i be concern lol

**Comment by thatskymod** [comment]
**Relevant threads:**
- https://github.com/pyinstaller/pyinstaller/issues/5854
- [Reddit Discussion on Antivirus Detection](https://www.reddit.com/r/antivirus/comments/nlubcn/installing_pyinstaller_windows_antivirus_detects/)

In summary, while the program itself is not malicious, the compilation framework PyInstaller used to create this executable is often abused  by bad actors to bundle malware. This malware can share signatures with legitimate code, leading to false positive detections by antivirus software.

For the time being, the PC port is excluded from the repository due to a few other issues that need to be resolved. One such issue is discussed here:
- https://github.com/thatskymod/Sky-CotL-Scripts/discussions/93

It is recommended to wait until these issues are resolved and a new update is released before proceeding.

**Reply by zerobest1123** [reply]
thank you!
got my heart stop for a sec lol

**Comment by thatskymod** [comment, marked as answer]
Sorry, this issue refers to a file not present in the repository. As such, further support on this matter won't be pursued. It is recommended to wait until a stable version of the file is published.
